features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/recorder/base64/Cache.kt

```diff
@@ -0,0 +1,17 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.sessionreplay.internal.recorder.base64
+
+import android.graphics.drawable.Drawable
+
+internal interface Cache {
+    fun put(drawable: Drawable, base64String: String)
+    fun get(key: Int): String?
+    fun size(): Int
+    fun clear()
+    fun isEmpty(): Boolean
```

Comment:
is it used?

Reply:
not used. I put it in as a possibly useful utility for the class, but I'll remove it until we actually need it